test-scripts/run-presto-kerberos-integration-test.sh

```diff
@@ -0,0 +1,66 @@
+#! /usr/bin/env bash
+# runs one or more Metabase test(s) against a Kerberized Presto instance
```

Comment:
I would do `set -o pipefail` at the very least so if something in the script fails the script itself will exit with nonzero

Reply:
Done in next commit.